tests/_data/templates/custom-i18n.php

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
@justlevine let's add one more test where the file name also contains non-ascii characters. 

I believe in that case the schema will still error as the fallback will still be an invalid GraphQL name 🤷🏻‍♂️ 

If it is invalid by time we `register_graphql_object_type()`, let's `graphql_debug()` and `continue;` instead of registering it.

Reply:
Just putting a `continue;` here punted the Fatal Error to querying for a bad template, so I extracted the formatting logic to a Utils method, and the `__typename` passed by the `Post` model now falls back to `DefaultTemplate` if the formatted name isnt compatible with GraphQL.